fetch.bs

```diff
@@ -2630,6 +2630,10 @@ boolean <var>http3Only</var>, run these steps:
    <li><p>If the user agent waits for full handshake completion to send the request, this interval
    includes the full TLS handshake even if other requests were sent using early data on
    <var>connection</var>.
+
+   <li><p>If the connection is an HTTP/3 <a>connection</a>,
+   <var>timingInfo</var>'s <a for="connection timing info">connection end time</a> should represent
+   the time right after the TLS handshake. [[!HTTP3]]
```

Comment:
I think HTTP/3 is pretty much the same as other versions of HTTP over TLS 1.3 in that if you send early data, you don't wait for the ServerHello?  If you want to use a term from the RFC for when all requests can be sent, I'd use 'Handshake Complete', which is the point at which the client has the final 1-RTT keys.

I'll note that early data can be rejected, and it might be worth clarifying whether the time the early data is initially sent should be used or after rejection.  I suspect the former, but it's worth being clear I think.

Note: 'handshare' is misspelled on line 2626.